image/templates/helm/stackrox-secured-cluster/templates/service-ca.yaml

```
@@ -13,4 +13,7 @@ metadata:
 type: Opaque
 stringData:
   ca.pem: |
+    {{- if hasKey (.Values.metadata).annotations "init-bundle.stackrox.io/name" }}
+    {{- fail "It looks like you're using the incorrect init-bundle format. Make sure that you're using an init-bundle that was formatted for the Helm installation method and not Operator installation method." | indent 4 }}
+    {{ end }}
```

Go templating has these different methods built in for controlling whitespace trimming:

```
{{ ... }}
{{- ... -}}
{{- ... }}
{{ ... -}}
```

Explanation for this, especially for control structures, can be found here https://helm.sh/docs/chart_template_guide/control_structures/#ifelse.

tl,dr: It seems to be a reasonable practice to always use `{{- ... }}` for line-based control structures like `if/else/end` in YAML. This is the method which usually seems to follow the principle of the least surprise by trimming exactly the amount of whitespace we would expect it to trim.

Hence, I would suggest you also use `{{- end }}` here, just like you used `{{- if ... }}` above.

```suggestion
    {{- end }}
```